decidim-assemblies/spec/controllers/assemblies_settings_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Assemblies
+    module Admin
+      describe AssembliesSettingsController, type: :controller do
+        routes { Decidim::Assemblies::AdminEngine.routes }
+
+        let(:organization) { create(:organization) }
+        let(:current_user) { create(:user, :confirmed, :admin, organization: organization) }
+        let!(:assemblies_setting) { create(:assemblies_setting, organization: organization) }
+
+        before do
+          request.env["decidim.current_organization"] = organization
+          request.env["decidim.current_assemblies_settings"] = assemblies_setting
```

Comment:
mmm, I didn't find any other place where you use this `env` entry. Is it really needed?